docs/libcurl/opts/CURLOPT_COOKIEFILE.3

```diff
@@ -60,6 +60,11 @@ option.
 
 Setting this option to NULL will (since 7.77.0) explicitly disable the cookie
 engine and clear the list of files to read cookies from.
+.SH SECURITY
+This document previously mentioned how specifying a non-existing file can also
+enable the cookie engine. While true, we strongly advice against using that
+method as it is simply too hard to be sure what file names that truly will
+remain non-existing in the long run.
```

Comment:
non-existent x2, and i'd nix that ie file names truly

Reply:
I rephrased